### ISSUE TITLE
Experiment Config

### ISSUE THREAD

**Issue by Wangpeiyi9979** [SapienzaNLP/spring#5]
Hi, thanks for your nice work.
I find there are several settings in the config
![image](https://user-images.githubusercontent.com/42565075/119466859-98f6c880-bd77-11eb-82e8-30c8004b1352.png)
It seems that the BFS PENMAN and BART baseline is in the same setting.
I wonder if I have missed any details

In addition,
following the script in the README.MD, I train the model and evaluate on the test dataset. 
Without using the `BLINK`, the model can achieve 84.4 smatch score.

I find in the config, the `use_recategorization` is `False`, so this result represents the result of `SPRINGDFS +silver` in the paper, right? But I do not find where the silver data is. It is strange.

**Comment by mbevila** [comment]
There was indeed an error in the config which I have now fixed. Now the settings are correct also for the commented out blocks.

Did you evaluate using the Damonte script? That one should be lower by ~0.3. Also, some fluctuations are expected given that the seed is not fixed -- probably your run was a lucky one. Do you replicate results with the pretrained checkpoint? 

We have not released the silver data.
